tf-modules/cumulus/variables.tf

```diff
@@ -248,6 +248,12 @@ variable "ecs_efs_config" {
   default     = null
 }
 
+variable "ecs_include_docker_cleanup_cronjob" {
+  description = "Flag to configure adding cronjob to clean up unused root space"
```

Comment:
Should we note this is an experimental option, or put a disclaimer on it given it's essentially a contribution?